etna/analysis/forecast/plots.py

```diff
@@ -430,7 +431,7 @@ def plot_backtest_interactive(
         for fold_number in folds:
             start_fold = fold_numbers[fold_numbers == fold_number].index.min()
             end_fold = fold_numbers[fold_numbers == fold_number].index.max()
-            end_fold_exclusive = pd.date_range(start=end_fold, periods=2, freq=ts.freq)[1]
+            end_fold_exclusive = timestamp_range(start=end_fold, periods=2, freq=ts.freq)[-1]
```

Comment:
Why is it -1 now?

Reply:
Because there are only 2 elements there with indices 0 and 1.  It has no difference.

Reply:
Then it is better to leave the old logic